PSReadLine/SamplePSReadLineProfile.ps1

```diff
@@ -600,3 +600,57 @@ Set-PSReadLineKeyHandler -Key RightArrow `
         [Microsoft.PowerShell.PSConsoleReadLine]::AcceptNextSuggestionWord($key, $arg)
     }
 }
+
+# Cycle through arguments on current line and select the text. This makes it easier to quickly change the argument if re-running a previously run command from the history
+# or if using a psreadline predictor. You can also use a digit argument to specify which argument you want to select, i.e. Alt+1, Alt+a selects the first argument
+# on the command line. 
+Set-PSReadLineKeyHandler -Key Alt+a `
+                         -BriefDescription SelectCommandArguments `
+                         -LongDescription "Set current selection to next command argument in the command line. Use of digit argument selects argument by position" `
+                         -ScriptBlock {
+    param($key, $arg)
+  
+    $line = $null
+    $cursor = $null
+    [Microsoft.PowerShell.PSConsoleReadLine]::GetBufferState([ref]$line, [ref]$cursor)
+  
+    $tokens = $null
+    $ast = $null
+    $parseErrors = $null
+    [Microsoft.PowerShell.PSConsoleReadLine]::GetBufferState([ref]$ast, [ref]$tokens, [ref]$parseErrors, [ref]$null)
+  
+    $asts = $ast.FindAll( {
+        $args[0] -is [System.Management.Automation.Language.StringConstantExpressionAst] -and
```

Comment:
Can you take into account `ConstantExpressionAst` as well? It currently skip a constant value like `100` when looping through the arguments.